### PR TITLE
Update admin.rst to include profile_path

### PR DESCRIPTION
**Pull request by hennna**
Update to include change made in #596 and #603 .

### CODE REVIEW

File: docs/configuration/overview/admin.rst [hunk 1]
```diff
@@ -9,13 +9,17 @@ Administration interface :ref:`operations documentation <operations_admin_interf
...
   }
 
 access_log_path
   *(required, string)* The path to write the access log for the administration server. If no
   access log is desired specify '/dev/null'.
 
+profile_path
+  *(optional, string)* The cpu profiler output path for the administration server. If no profile 
+  path is specified, the default is '/var/log/envoy/envoy.prof'.
 address
```
Comment (mattklein123) [inline docs/configuration/overview/admin.rst]: nit: newline before this line.

File: docs/configuration/overview/admin.rst [hunk 2]
```diff
@@ -9,13 +9,16 @@ Administration interface :ref:`operations documentation <operations_admin_interf
...
   }
 
 access_log_path
   *(required, string)* The path to write the access log for the administration server. If no
   access log is desired specify '/dev/null'.
-
+profile_path
+  *(optional, string)* The cpu profiler output path for the administration server. If no profile
+  path is specified, the default is '/var/log/envoy/envoy.prof'.
 address
```
Comment (mattklein123) [inline docs/configuration/overview/admin.rst]: Sorry I meant please keep a blank line between attributes as per the rest of the documentation.

Comment (hennna) [inline docs/configuration/overview/admin.rst]: Sorry. I should have noticed that.

File: docs/configuration/overview/admin.rst [hunk 3]
```diff
@@ -9,13 +9,18 @@ Administration interface :ref:`operations documentation <operations_admin_interf
 
   {
     "access_log_path": "...",
+    "profile_path": "...",
     "port": "..."
```
Comment (RomanDzhabarov) [inline docs/configuration/overview/admin.rst]: sorry, not your change, but can you fix that to `address`

**Pull request merged**